test-suite/paralleltestrunner.hpp

```diff
@@ -217,16 +216,17 @@ int main( int argc, char* argv[] )
                 // check for number of processes
                 std::vector<std::string> tok;
                 boost::split(tok, arg, boost::is_any_of("="));
-                if (tok.size() == 2 && tok[0] == "--nProc") {
+                if (tok[0] == "--nProc" && tok.size() == 2) {
```

Comment:
Is it guaranteed that tok has positive size?  The order of the conditions seemed to enforce that.

Reply:
There's always at least one element in tok, even if arg is empty or does not contain any "=".